lib/handlers/unsubscribe.js

```diff
@@ -68,7 +68,7 @@ function completeUnsubscribe (err) {
     return client.emit('error', err)
   }
 
-  if (!packet.close && packet.unsubscriptions.length > 0) {
+  if (client.clean === true && packet.unsubscriptions.length > 0) {
```

Comment:
the check for `packet.close`  must stay. See https://github.com/mcollina/aedes/blob/275bda32461ea8053c8610b8a3e5b6f0f838f1b6/lib/client.js#L214.
Basically if `packet.close === true && client.clean === false` we should __not__ emit.

Reply:
@mcollina , thank you for a clarification, please see the new commit